airdrop/application/services/user_registration_services.py

```diff
@@ -9,6 +9,56 @@
 
 class UserRegistrationServices:
 
+    def eligibility(self, inputs):
+
+        status = HTTPStatus.BAD_REQUEST
+
+        try:
+            schema = {
+                "type": "object",
+                "properties": {
+                    "airdrop_window_id": {"type": "string"},
+                    "airdrop_id": {"type": "string"},
+                    "address": {"type": "string"},
+                    "signature": {"type": "string"},
+                },
+                "required": ["signature", "address", "airdrop_id", "airdrop_window_id"],
+            }
+
+            validate(instance=inputs, schema=schema)
+
+            airdrop_id = inputs["airdrop_id"]
+            airdrop_window_id = inputs["airdrop_window_id"]
+            address = inputs["address"].lower()
+            signature = inputs["signature"]
+
+            airdrop_window = self.get_user_airdrop_window(
+                airdrop_id, airdrop_window_id
+            )
+
+            if airdrop_window is None:
+                raise Exception(
+                    "Airdrop window is not accepting registration at this moment"
+                )
+
+            is_eligible_user = self.check_user_eligibility(
```

Comment:
Why are we hard coding AGIX here ?

Reply:
Removed hardcoded value AGIX, the actual implementation will update later, for now the eligibility is always `true`